src/DynamoDbServiceProvider.php

```diff
@@ -40,11 +40,18 @@ protected function bindForApp($marshalerOptions = [])
             $config = [
                 'credentials' => [
                     'key' => config('services.dynamodb.key'),
-                    'secret' => config('services.dynamodb.secret'),
+                    'secret' => config('services.dynamodb.secret')
```

Comment:
It does still work, however I don't believe a parameter we know may or may not be set should be passed through.

I can make the change if you wish, as it doesn't affect functionality.